plugins/modules/image.py

```diff
@@ -95,6 +95,17 @@
         editable=True,
         description=["A list of customized tags of this new Image."],
     ),
+    "regions_to_replicate": SpecField(
```

Comment:
nitpick: We could maybe name this field something like `replica_regions` or `replicate_to`?

Reply:
Good point! I want to keep the word `region` in the naming because it's what the endpoint calls it. Probably `replica_regions` works better in this case. I can also change this attribute name in TF to keep it consistent for user: https://github.com/linode/terraform-provider-linode/blob/proj/image-gen2/docs/resources/image.md?plain=1#L64

Reply:
If we do that can we leave a comment with the actual field name? I know its obvious now but when i was going through the Docs for one of the PRs it did throw me off some i kept thinking I had the wrong endpoint.

Reply:
Thank you both for the suggestion! Just updated the name and added a comment.